package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "music-library",
+  "version": "1.0.0",
+  "description": "---",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha tests/**/*.js --exit --recursive --timeout 60000 --file ./tests/test-setup.js",
+    "pretest": "node utils/create-database.js test",
+    "posttest": "node utils/drop-database.js",
+    "prestart": "node utils/create-database.js",
+    "start": "nodemon -r dotenv/config index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.17.2",
+    "mysql2": "^2.3.3"
+  },
+  "devDependencies": {
+    "dotenv": "^16.0.0",
+    "nodemon": "^2.0.15"
```

Comment:
I couldnt run the tests as there are some important dependencies missing here (chai, mocha and supertest). After installing these I could run the tests, guessing you have these in your local repo but forgot to push them up?

Reply:
Slightly confused where these have gone. I have been running npm start and getting it to pass. 
I have checked my package.json on my local and they are not in there. Will add them in and run the test again and re-commit to gut hub.